src/main/java/racingcar/controller/RacingController.java

```diff
@@ -0,0 +1,35 @@
+package racingcar.controller;
+
+import static racingcar.constants.Symbol.ZERO;
+
+import racingcar.model.Cars;
+import racingcar.model.GameRound;
+import racingcar.view.InputView;
+import racingcar.view.OutputView;
+
+public class RacingController {
+    private final InputView inputView;
+    private final OutputView outputView;
+
+    public RacingController(InputView inputView, OutputView outputView) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+    }
+
+    public void run() {
+        String rawCarsName = inputView.inputCarNames(); // carName을 입력 받는다.
+        Cars cars = new Cars(rawCarsName);
+        String count = inputView.inputTrialCount(); // 시도 횟수를 입력 받는다.
+        GameRound gameRound = new GameRound(count);
+        startRacingGame(cars, gameRound); // 게임을 시작한다.
+        outputView.printWinner(cars); // 우승자를 출력한다.

```

Comment:
블로그에서 이 부분에 대해 고민이 되신다고 올리신 글을 봤습니다
저는 이 winner를 출력하는 부분은 단순히 우승자의 **이름**만 전달하면 된다라고 생각합니다.
따라서 cars 객체를 전체다 보낼 필요가 없다고 생각합니다.
```suggestion
List<String> winners = cars.getWinners();
outputView.printWinner(winners)
```
요런식으로 String 리스트를 보내서 view에서는 이름을 출력만 하도록 하는게 어떨까 합니다

Reply:
좋은 제안 너무나도 감사합니다..!
저도 이렇게 넘겨주는 게 MVC 패턴에 더 적합하다고 생각이 드네요..